cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -334,16 +334,9 @@ def configure_ua(token, enable=None):
     # related. We can distinguish them by checking if `service` is non-null
     # or null respectively.
 
-    # pylint: disable=import-error
-    from uaclient.messages import ALREADY_ENABLED
-
-    # pylint: enable=import-error
-
-    UA_MC_ALREADY_ENABLED = ALREADY_ENABLED.name
-
     enable_errors: List[dict] = []
     for err in enable_resp.get("errors", []):
-        if err["message_code"] == UA_MC_ALREADY_ENABLED:
+        if err["message_code"] == "service-already-enabled":
```

Comment:
@orndorffgrant Will this hard-coding of this string hit is an an unknown dependency on uaclient if those error codes change in the future, or are we guaranteed backwards-compatibliity with error message_codes in all future releases?

Reply:
Unless we make a mistake, message codes will not change. So if this were to break it would be an ubuntu-pro-client bug, not a cloud-init bug.

In the future, ubuntu-pro-client should provide an api function for enabling that you can import. That would come with defined exceptions that you can catch instead of looking at json error objects.